test/bigtest/network/scenarios/fetch-job-profiles-success.js

```diff
@@ -1,28 +1,28 @@
-const tempData = [{
+export const sampleJobExecution = {
   id: 'de96f7a1-8706-42e9-ba86-c44c44592511',
   hrId: 3,
-  parentJobId: 'de96f7a1-8706-42e9-ba86-c44c44592511',
-  subordinationType: 'PARENT_SINGLE',
   jobProfileInfo: {
     id: '558dede0-515d-11e9-8647-d663bd873d93',
     name: 'default',
-    dataType: 'MARC',
   },
-  sourcePath: 'EBOOKS1.mrc',
   fileName: 'EBOOKS1.mrc',
   runBy: {
     firstName: 'John',
     lastName: 'Doe',
   },
   progress: {
-    jobExecutionId: 'de96f7a1-8706-42e9-ba86-c44c44592511',
     current: 21650,
     total: 45265,
   },
   startedDate: '2020-01-27T10:10:18.743+0000',
-  status: 'PARSING_IN_PROGRESS',
-  uiStatus: 'RUNNING',
+  completedDate: '2019-10-30T12:40:01.000',
```

Comment:
Could we have completedDate field if the job is in progress (According to the status in line 19)?

Reply:
Good point. In this particular case, it makes sense to remove it.